twake/backend/node/test/e2e/workspaces/workspaces.invite-tokens.spec.ts

```diff
@@ -0,0 +1,513 @@
+
+      const jwtToken = await platform.auth.getJWTToken({ sub: userIdFromAnotherWorkspace });
+
+      const response = await platform.app.inject({
+        method: "DELETE",
+        url: `${url}/companies/${companyId}/workspaces/${workspaceId}/users/tokens/123`,
+        headers: { authorization: `Bearer ${jwtToken}` },
+        payload: {},
+      });
+      expect(response.statusCode).toBe(403);
+
+      done();
+    });
+
+    it("should 204 when user is a workspace member", async done => {
+      const workspaceId = testDbService.workspaces[0].workspace.id;
+      const userId = testDbService.workspaces[0].users[0].id;
+      const companyId = testDbService.company.id;
+
+      await workspaceServicesAPI.workspaces.createInviteToken(companyId, workspaceId);
+
+      const jwtToken = await platform.auth.getJWTToken({ sub: userId });
+
+      const beforeResponse = await platform.app.inject({
+        method: "GET",
+        url: `${url}/companies/${companyId}/workspaces/${workspaceId}/users/tokens`,
+        headers: { authorization: `Bearer ${jwtToken}` },
+      });
+
+      expect(beforeResponse.statusCode).toBe(200);
+
+      const response = await platform.app.inject({
+        method: "DELETE",
+        url: `${url}/companies/${companyId}/workspaces/${workspaceId}/users/tokens/${
+          beforeResponse.json().resources[0].token
+        }`,
+        headers: { authorization: `Bearer ${jwtToken}` },
+      });
+
+      expect(response.statusCode).toBe(204);
+
+      const afterResponse = await platform.app.inject({
+        method: "GET",
+        url: `${url}/companies/${companyId}/workspaces/${workspaceId}/users/tokens`,
+        headers: { authorization: `Bearer ${jwtToken}` },
+      });
+
+      expect(afterResponse.statusCode).toBe(404);
+
+      done();
+    });
+
+    it("should 404 when deleting token that doesn't exists", async done => {
+      const workspaceId = testDbService.workspaces[0].workspace.id;
+      const userId = testDbService.workspaces[0].users[0].id;
+      const companyId = testDbService.company.id;
+
+      const jwtToken = await platform.auth.getJWTToken({ sub: userId });
+
+      const response = await platform.app.inject({
+        method: "DELETE",
+        url: `${url}/companies/${companyId}/workspaces/${workspaceId}/users/tokens/123456`,
+        headers: { authorization: `Bearer ${jwtToken}` },
+        payload: {},
+      });
+      expect(response.statusCode).toBe(404);
+
+      done();
+    });
+  });
+
+  describe("Join workspace using token", () => {
+    let workspaceId;
+    let userId;
+    let companyId;
+    let inviteToken;
+
+    beforeAll(async () => {
+      await startup();
+      companyId = testDbService.company.id;
+      workspaceId = testDbService.workspaces[0].workspace.id;
+      userId = testDbService.workspaces[2].users[0].id;
+      inviteToken = await workspaceServicesAPI.workspaces.createInviteToken(companyId, workspaceId);
```

Comment:
Your current issue probably comes from here, now you have a list and so it doesn't match the schema that wait for a single string and not a list of object. So you get a 400 error. @romanesko